tomls/omnibus-arbitrum-sepolia/spot/dai.toml

```diff
@@ -0,0 +1,80 @@
+[invoke.createDaiSynth]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "owner"
+func = "createSynth"
+args = ["Synthetic DAI", "sDAI", "<%= settings.owner %>"]
+extra.synth_dai_market_id.event = "SynthRegistered"
+extra.synth_dai_market_id.arg = 0
+extra.synth_dai_token_address.event = "SynthRegistered"
+extra.synth_dai_token_address.arg = 1
+
+[invoke.setSynthPriceDai]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "updatePriceData"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    "<%= extras.dai_oracle_id %>",
+    "<%= extras.dai_oracle_id %>",
+    "<%= settings.spotStrictStalenessTolerance %>",
+]
+
+[invoke.setSynthMaxMarketCollateralDai]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+fromCall.args = []
+func = "configureMaximumMarketCollateral"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    "<%= imports.dai_mock_collateral.contracts.MintableToken.address %>",
+    "<%= settings.synthDaiMaxMarketCollateral %>",
+]
+
+[invoke.configureSynthWrapperDai]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "setWrapper"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    "<%= imports.dai_mock_collateral.contracts.MintableToken.address %>",
+    "<%= settings.synthDaiMaxMarketCollateral %>",
+]
+
+[invoke.addSynthSettlementStrategyDai]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_dai_market_id %>"]
+func = "addSettlementStrategy"
+args = [
+    "<%= extras.synth_dai_market_id %>",
+    { strategyType = "1", settlementDelay = "0", settlementWindowDuration = "1", priceVerificationContract = "0x0000000000000000000000000000000000000000", feedId = "0x0000000000000000000000000000000000000000000000000000000000000000", url = "", settlementReward = 0, minimumUsdExchangeAmount = "0", maxRoundingLoss = "0", priceDeviationTolerance = "0", disabled = false },
```

Comment:
Should at least verify maxRoundingLoss = 0 is good here.

Reply:
Deploy is successful. Then it is set to non zero with next invoke

Reply:
yep makes sense.  should be a very small value, def not 0 which i think is being set in the next invoke 👍🏽

Reply:
I've set it to `1` minimal non-zero value